src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends FigCol {
```

Comment:
The class name 'FigCol' suggests that it might be an abstract class providing common behavior for figures, which is discouraged according to the checklist. Consider using an interface to define common behavior and ensure that 'FigCol' is not an abstract class that sets behavior for its subclasses.